app/forms/date_form.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import calendar
+
+from wtforms import Form, FormField, SelectField, StringField
+from wtforms import validators
+
+from app.validation.validators import DateCheck, DateRangeCheck, DateRequired, MonthYearCheck
+
+logger = logging.getLogger(__name__)
+
+
+def get_date_form(answer=None, to_field_data=None, validate_range=False, error_messages=None):
```

Comment:
I've tried this and the short answer is it's a metaclass so behaves differently to standard class init and doesn't work.